index.html

```diff
@@ -105,19 +111,28 @@
     </section>
 
     <section>
-      <h2>Payment Method Identifier</h2>
-      <p>
-        The <dfn>Payment Method Identifier</dfn> is a string that uniquely identifies a payment method that a user can use to complete a transaction. For example, Visa, MasterCard, and American Express are payment methods used in some countries.
-      </p>
+      <h2>URLs as Payment Method Identifiers</h2>
+      <p>When the party
+	responsible for a payment method wishes to publish
+	machine-readable information associated with the payment
+	method, it does so with a URL.
+	This URL approach offers at least two benefits: identifier
+	decentralization and origins as a trust mechanism.
+	In particular, owners of proprietary payment methods
+	can use Web servers under their control to publish
+	security information about authorized payment apps.</p>
+      <div class="issue" title="Payment method manifest specification in progress">The Web Payments Working Group is developing a payment method manifest specification for the information a payment method owner would want to publish. Payment method identifiers that are URLs will help user agents locate these manifest files, although the exact mechanism is not yet defined. See the early <a href="https://github.com/zkoch/zkoch.github.io/blob/master/payment-manifest.md">Payment Manifest Proposal</a>.
+	</div>
+      <div class="issue" title="URL maintenance and performance">Either in the payment method manifest specification or in separate "good practice" documentation, the Working Group should address issues of server performance for payment method identifiers that are URLs. Furthermore, when parties feel they do not need to publish a manifest, we should explain that a manifest with an explicit machine-readable assertion that "any payment app may implement" is preferable to lack of a manifest.
+	</div>
       <section>
-        <h2>Identifier format</h2>
-        <p>Payment method identifiers are strings containing a <a>URL</a>. The string MUST be an <a>absolute URL</a>.</p>
+        <h2>Syntax</h2>
+	<p>When URLs are used for payment method identifiers they MUST
+	  be absolute URLs including only the schema, origin, and path parts. The URL scheme MUST be <code>https</code>. These URLs MUST NOT include query string parameters or fragment identifiers.</p>
       </section>
       <section>
-        <h2>Identifier equivalence</h2>
-        <p>When the PaymentRequest API is invoked, the web page provides a list of identifiers for supported payment methods.
-        The user agent must compare these identifiers to those available to the user and use this to filter what the user
-        can select. To determine whether two identifiers match, perform the following test:</p>
+        <h2>Matching</h2>
```

Comment:
> Similarly, I don't think this specification should say what to do when there is, for example a 404 on a manifest file.

I think we'll need to add that when we add the manifest stuff to this spec but happy to do that later when @maxpassion has this ready.

> I think that the PR API spec should speak to this behavior but not the PMI spec.

There seems to be a lot of work still required on the `show()` algorithm to call out to the PMI matching now that we have that defined. https://w3c.github.io/browser-payment-api/#show-method

Happy to move it there but we should probably give the caller some indication that method identifiers have been removed (Step 8) /cc @adrianba @zkoch @rvm4